astropy/utils/console.py

```diff
@@ -86,6 +40,33 @@ class PyreadlineConsole(object): pass
 _DEFAULT_ENCODING = 'utf-8'
 
 
+def _get_out_stream():
+    try:
+        from IPython import get_ipython
+    except ImportError:
+        pass
+    try:
```

Comment:
This `try` looks redundant because it only catches the `NameError`. Is there any motivation behind this?